src/integrationtests/resources/com/aws/greengrass/integrationtests/status/fss_service_10s_recovery.yaml

```diff
@@ -33,7 +33,7 @@ services:
         startup:
           requiresPrivilege: true
           script: |-
-            powershell -command "& { if (Test-Path ErrorIndicator) {  ping -n 7 127.0.0.1 > NUL; echo "Startup Failed"; exit 1; } }"
+            powershell -command "& { if (Test-Path ErrorIndicator) { Start-Sleep -Milliseconds 7000; echo "Startup Failed"; exit 1; } }"
```

Comment:
Why change this? Using powershell significantly slows down the test. Using ping is equivalent to a sleep